en/components/grids_templates/summaries.md

```diff
@@ -534,7 +432,115 @@ When a default summary is defined, the height of the summary area is calculated
 
 }
 
+## Disable Summaries
+
+The [`disabledSummaries`]({environment:angularApiUrl}/classes/@@igTypeDoc.html#disabledSummaries) property provides precise control over the summary functions displayed for each column in the grid. This property is particularly useful when you want to remove unnecessary or less relevant summaries, ensuring that only meaningful and context-specific results are presented to users. It accepts an array of strings, such as `['count', 'min', 'max']`, where each string corresponds to a **summary key** that should be excluded from the column's displayed summaries.
```

Comment:
property provides precise control over the summary functions displayed for each column in the grid. - this may paraphrase as: property provides precise per column control over the Ignite UI for Angular grid summary feature. This property enables the user to customize the summaries,...

In general: Ignite UI *framework* grid *feature* - when mentioning grid features, we need to avoid mentioning only the feature name without "grid", users are searching for: angular grid summaries or grid summaries